opencti-platform/opencti-graphql/src/database/file-storage.js

```diff
@@ -334,17 +334,17 @@ export const deleteAllFiles = async (context, user, path) => {
   return deleteFiles(context, user, ids);
 };
 
-const resolveImageFiles = async (context, user, fileNodes, resolveEntity) => {
+const resolveImageFiles = async (context, user, files, resolveEntity) => {
   const elasticFiles = resolveEntity.x_opencti_files;
-  return fileNodes.map((file) => {
-    const elasticFile = (elasticFiles ?? []).find((e) => e.id === file.node.id);
+  return files.map((file) => {
+    const elasticFile = (elasticFiles ?? []).find((e) => e.id === file.id);
     if (elasticFile) {
       return {
         ...file,
         node: {
-          ...file.node,
+          ...file,
```

Comment:
I don't think there should be a `node` here, since we call this method before transforming files to nodes

Reply:
fixed